replication-api-impl/pom.xml

```diff
@@ -331,12 +331,12 @@
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.54</minimum>
+                                            <minimum>0.43</minimum>
```

Comment:
✏️ Definitely not going in the right direction

Reply:
The code that I've added is 100% covered but this is still going down because the code that we had before was still mostly tested but there was a lot more of it. I could add tests for random classes in this PR but I think that would be better left for another one. I want to increase our test coverage too, maybe we should bring it up at our next planning meeting.